downloads/swan-lake-release-notes/swan-lake-alpha/RELEASE_NOTE.md

```diff
@@ -0,0 +1,1555 @@
+        }
+
+    remote function onUpdateMessage(UpdateMessage msg) returns Acknowledgement|UpdateMessageError {
+        // implement action to execute on content-update for topic
+        }
+
+    remote function onSubscription(Subscription msg) returns SubscriptionAccepted|SubscriptionPermanentRedirect|
+        SubscriptionTemporaryRedirect|BadSubscriptionError|InternalSubscriptionError {
+        // implement action to execute on new subscription
+        }
+
+    remote function onSubscriptionValidation(Subscription msg) returns SubscriptionDeniedError? {
+        // implement action to execute on subscription validation
+        }
+
+    remote function onSubscriptionIntentVerified(VerifiedSubscription msg) {
+        // implement action to execute on subscription intent verification
+        }
+
+    remote function onUnsubscription(Unsubscription msg) returns UnsubscriptionAccepted|BadUnsubscriptionError|
+        InternalUnsubscriptionError {
+        // implement action to execute on unsubscription
+        }
+
+    remote function onUnsubscriptionValidation(Unsubscription msg) returns UnsubscriptionDeniedError? {
+        // implement action to execute on unsubscription validation
+        }
+
+    remote function onUnsubscriptionIntentVerified(VerifiedUnsubscription msg) {
+        // implement action to execute on unsubscription intent verification
+        }
+    }    
+    ```
+
+**Hub Client Implementation**
+
+- The `websubhub:Client` is introduced to distribute the updated content to subscribers.
+- The example below is a sample use-case of the WebSub Hub Client.
+
+```ballerina
+    service / websubhubon hubListener {
+    remote function onSubscriptionIntentVerified(websubhub:VerifiedSubscription msg) {
+
+            // you can pass client config if you want 
+            // say maybe retry config
+            websub:HubClient hubclient = new (msg);
+            check start notifySubscriber(hubclient);
+        }
+
+        function notifySubscriber(websubhub:HubClient hubclient) returns error? {
+            while (true) {
+                // fetch the message from MB
+                check hubclient->notifyContentDistribution({content: "This is sample content delivery"});
+            }
+        }
+    }
+```
+
+**Publisher Implementation**
+
+The `websub:PublisherClient` is moved to the `websubhub:PublisherClient`.
+
+##### Removed Modules
+
+###### Config
+
+The configuration use cases are now covered under the `configurable` language feature. 
+
+###### Math
+
+The APIs related to random number generation were moved to the new `random` module. The rest of the APIs have replacements in the `lang.float` and `lang.int` packages.
+
+###### Stringutils
+
+The regex-related APIs that were supported by this module have been moved to the new `regex` module. The rest of the APIs have replacements in the langlib packages.
+
+###### Socket
+
+The `socket` module was removed and got replaced by the `TCP` and `UDP` modules.
+
```

Comment:
```suggestion

```

Reply:
+1 , this is already mentioned under `UDP/TCP module improvements`

Reply:
+1